internal/pkg/source/entry_test.go

```diff
@@ -26,31 +27,55 @@ func TestParseLogEntryDefault(t *testing.T) {
 
 			assert.Equal(t, "Hello World", fieldKindToValue[config.FieldKindMessage], fieldKindToValue)
 			assert.Equal(t, "-", fieldKindToValue[config.FieldKindLevel], fieldKindToValue)
-			assert.Equal(t, "-", fieldKindToValue[config.FieldKindTime], fieldKindToValue)
+			assert.Equal(t, "-", fieldKindToValue[config.FieldKindNumericTime], fieldKindToValue)
 		},
 	}, {
 		Name: "time_number",
 		JSON: `{"time":1}`,
 		Assert: func(tb testing.TB, fieldKindToValue map[config.FieldKind]string) {
 			tb.Helper()
 
-			assert.Equal(t, "1", fieldKindToValue[config.FieldKindTime], fieldKindToValue)
+			assert.Equal(t, time.Unix(1, 0).Format(time.RFC3339), fieldKindToValue[config.FieldKindNumericTime], fieldKindToValue)
```

Comment:
```suggestion
			assert.Equal(t, time.Unix(1, 0).Format(time.RFC3339).UTC(), fieldKindToValue[config.FieldKindNumericTime], fieldKindToValue)
```

and in another places